cmd/cmount/mount.go

```diff
@@ -53,28 +52,9 @@ func mountOptions(VFS *vfs.VFS, device string, mountpoint string, opt *mountlib.
 		options = append(options, "-o", "debug")
 	}
 
-	// Determine if ExtraOptions already has an opt in
```

Comment:
Not sure why you've removed this code - the `hasExtraOption` works out if that option has been specified so it doesn't override the one supplied by the user.

Reply:
If user supplies any options, those are added at the end. From my testing the last one wins. I can even add the same option multiple times as, e.g. `-o gid=11 -o gid=-1` and the resulting mount will use the last -1 value. The same is indicated here: https://github.com/billziss-gh/winfsp/issues/204#issuecomment-441845116.

I noticed the function were introduced after a [discussion](https://forum.rclone.org/t/issue-with-rclone-mount-and-resilio-sync/14730/28), but did not see any "proof" that it actually made a difference.

The reason for not just keeping it was that I found that you can specify multiple options, such as both uid and gid, in the same comma separated string. So both `-o uid=-1 -o gid=11` and `-o uid=-1,gid=11` have the same result. But the existing hasExtraOptions check did not detect this second case. I added a commit to fix this, but then realized there were still weaknesses. And then, what about the other options that rclone sets by default? "--FileSystemName", "--VolumePrefix" and "-o volname"? There is no check if these are set by user..

So I concluded it is better to rely on WinFsp handling this correctly, instead of trying to catch all corner cases ourselves.

Tried to explain in the section "Continuing from previous point, but more generally", but totally understand it is easy to miss in that all too long description! (darn it, this comment ended up quite long as well..)